create/create.go

```diff
@@ -0,0 +1,41 @@
+package create
+
+import (
+	"fmt"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/tarantool/cartridge-cli/create/templates"
+	"github.com/tarantool/cartridge-cli/project"
+)
+
+func CreateProject(projectCtx *project.ProjectCtx) error {
+	log.Infof("Creating an application %q...", projectCtx.Name)
+
+	// check that application doesn't exist
+	if _, err := os.Stat(projectCtx.Path); err == nil {
```

Comment:
That if you e.g. don't have enough permission to open this dir? It returns an error but you ignore it